modules/detection/DetectionEvents.js

```diff
@@ -0,0 +1,44 @@
+/** Event triggered by NoAudioSignalDetector when the local audio device associated with a JitsiConference goes silent
```

Comment:
Well If we do this file I'm guessing we would have to move all the detection related files there. is that acceptable?

Reply:
Most of the events are defined in the service directory. We keep the implementation in modules.

But since we want to ship the PR sooner, let's address this later.